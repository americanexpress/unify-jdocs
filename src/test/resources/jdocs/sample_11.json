{
  "members": [
    {
      "ref": "ref1",
      "name": "deepak"
    }
  ]
}
