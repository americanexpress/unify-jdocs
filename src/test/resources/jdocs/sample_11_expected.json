{
  "members": [
    {
      "ref": "ref1",
      "name": "deepak"
    },
    {
      "ref": "ref1",
      "name": "deepak"
    },
    {
      "ref": "ref1",
      "name": "deepak"
    }
  ]
}
